### DISCUSSION TITLE
加载插件路径

### DISCUSSION THREAD

**Discussion by MangMax** [zouyaoji/vue-cesium#397] [Ideas]
我看到在vue-ceium中使用超图或者EarthSDK这些需要直接修改cesiumPath，有没有可能也和Mars3D一样选择要引入的资源或者资源路径，比如我想使用超图的3d服务，下载配置这个https://github.com/SuperMap/iClient3D-for-WebGL/tree/main/Cesium_S3MLayer_Plugins/S3MTilesLayer/Build路径即可使用

**Comment by zouyaoji** [comment]
mars3d是因为mars3d库自身这样设计了。

超图的似乎没有吧。目前定制viewer的初始化方式可以尝试，vc-viewer上的viewerCreator属性。
